### ISSUE TITLE
devices.yml and multiple topics

### ISSUE THREAD

**Issue by rhamblen** [sgupta999/mqtt-bridge-smartthings#6]
Im writing a device for a custom item.
Im trying to put it the device and everything crashes...

Format I thought was...
Device Name:
   attriubte: xxx
       subscribe:
          topic:
          topic:
          topic:

however in V1.0 of the code I got an error when I add a custom attribute and topic and it read the second topic in the list.
Using the new code it appears to work!  
 Am I right to assume you fixed the problem!!!

**Comment by sgupta999** [comment]
My goal was that nothing should crash - so not doing a good job there.

I did fix a bunch of issues that were causing crashes because of the state corruption.


I think you might have the format wrong - 

I think it is

Device 
  Publish / subscribe 
     Attribute 
          Topic
               Command
          Topic etc

**Comment by sgupta999** [comment]
My most heavily commented example in devices.example was wrong as another user commented so u fixed that too 

Make sure the device matches one of the capability items in the capability map in mbs-SmartApp lite or full - else just create one on your own if you really want to get adventurous

**Comment by rhamblen** [comment]
before I read your update I tested withthe new code and  it did fix things. Everything worked as I expected it to origionally.
I created a new device in the smartapp and i had the subscription work to the devices i created in devices.yml
I now have the power and energy information transferring between the shelly 4pro and smartthings.
I'll write a device handler tonight and post the information.   

We have the devices working for shelly switches with default firmware and giving the power information. Yippee
